doc/radosgw/qat-accel.rst

```diff
@@ -69,19 +69,75 @@ hardware is available and QAT is capable to handle it.
 
 Configuration
 =============
-1. QAT based Encryption for RGW 
-
-Edit the Ceph configuration file to make use of QAT based crypto plugin::
+1. Prerequisites
+
+Make sure the QAT driver with version v1.7.L.4.14.0 or higher has been installed.
+Remember to set an environment variable "ICP_ROOT" for your QAT driver package
+root directory. 
```

Comment:
```suggestion
#. Prerequisites

   Make sure the QAT driver with version v1.7.L.4.14.0 or higher has been installed.
   Remember to set an environment variable "ICP_ROOT" for your QAT driver package
   root directory. 
```

@liumiaomiaoIntel i meant to suggest something like this ^ . see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html for more details on indent in reStructuredText.

Reply:
Thanks for reminding. I have some changes based on your suggestion. Please review it.